### PR TITLE
added arrow, image, upload, Navbar Component

### PR DESCRIPTION
**Pull request by Dulinag**
- Frontend framework developed

### CODE REVIEW

**Comment by jubuyer** [comment]
- Dockerized frontend
  - Added service to docker-compose
 
How to spin up docker services:
```bash
docker-compose up --build -d # spins up docker services
docker ps # check running docker containers
```

Relevant paths for services:
Frontend - localhost:3000
Backend - localhost:8080
MongoDB - localhost:27017
MongoExpress - localhost:8081

**Comment by EricLi-Dev** [comment]
[Frontend](localhost:3000)
[Backend](localhost:8080/api/home)
[MongoDB](localhost:27017)
[MongoExpress](localhost:8081)

File: client/intial/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// import React, {useEffect, useState} from 'react'
```
Comment (EricLi-Dev) [inline client/intial/index.js:1]: Can we remove these comments?

**Review by EricLi-Dev — APPROVED** [review]
lgtm

**Review by jubuyer — APPROVED** [review]
Changes look good to merge.

**Comment by jubuyer** [comment]
Closes #18 (navbar added),
Closes #19 (image section added),
Closes #20 (arrow below hero text added),
Closes #21 (upload section added)

**Pull request merged**